scripts/check_doc_links.py

```diff
@@ -56,6 +56,7 @@
     "https://gateway.autonolas.tech/ipfs/<hash>,",  # non link (400)
     "https://github.com/valory-xyz/open-autonomy/trunk/infrastructure",  # svn link (404)
     "http://host.docker.internal:8545",  # internal (ERR_NAME_NOT_RESOLVED)
+    "https://notes.ethereum.org/@vbuterin/eip-1559-faq",  # temporarily down
```

Comment:
cab3399662a2b2210c66c007c04499757ff632c3 can be reverted before merging on `main`.